docs/ReleaseGuide.md

```diff
@@ -0,0 +1,24 @@
+## Guide to release new version of matlab-bindings
```

Comment:
Here you have to add a note to update `Contents.m` (see changes in https://github.com/precice/matlab-bindings/pull/27/files)